src/prefect/server/models/saved_searches.py

```diff
@@ -30,12 +31,11 @@ async def create_saved_search(
         saved_search (schemas.core.SavedSearch): a SavedSearch model
 
     Returns:
-        db.SavedSearch: the newly-created or updated SavedSearch
-
+        orm_models.SavedSearch: the newly-created or updated SavedSearch
     """
 
     insert_stmt = (
-        db.insert(db.SavedSearch)
+        db.insert(orm_models.SavedSearch)
```

Comment:
Why do we still need the database abstraction here -- something about the insert?